docs/source/user-guide/cli/installation.md

```diff
@@ -56,8 +56,7 @@ this to work.
 
 ```bash
 git clone https://github.com/apache/datafusion
-cd arrow-datafusion
-git checkout 12.0.0
```

Comment:
When i tried to check out 12.0.0. tag, the docker build command failed with some unsatisfied dependencies.